qiskit/tools/__init__.py

```diff
@@ -18,15 +18,19 @@
 .. currentmodule:: qiskit.tools
 
 Parallel Routines
-=================
+-----------------
+
+A helper function for calling a custom function with python map
```

Comment:
This line is very vague and doesn't really help me understand the purpose of the Parallel Routines module and when I might want to use it. Could you add a bit more detail?